Jamfile

```diff
@@ -505,7 +505,7 @@ feature export-extra : off on : composite propagated ;
 variant test_release : release
 	: <asserts>production <debug-symbols>on
 	  <invariant-checks>full <boost-link>shared <optimization>off
-	  <export-extra>on <debug-iterators>on <threading>multi
+	  <export-extra>on <debug-iterators>on <threading>multi <inlining>on
```

Comment:
I'm not entirely sure why this turned out to be a problem now, but basically there used to be a bug in boost-build where it passed on ``-finline-functions`` to clang when ``<inlining>full``. This was fixed last year, but hasn't propagated to the package on travis. In combination with this, clang used to ignore unknown arguments, but recently started to print a warning. Since I build with ``-Werror``, that warning turned into an error